bindings/python/pymongoarrow/schema.py

```diff
@@ -62,8 +63,13 @@ def _normalize_mapping(mapping):
 
     def _get_projection(self):
         projection = {"_id": False}
-        for fname, _ in self.typemap.items():
-            projection[fname] = True
+        for fname, ftype in self.typemap.items():
+            if isinstance(ftype, StructType):
```

Comment:
Nice, do we need to do the same for a fields that's a list_() of structs?

Reply:
Something that looks like {list: [{a: ...}, {b: ...}, {c: ...}]}?

Reply:
It'll be like `Schema({"list": list_(pyarrow.struct([field("a", int32())]))})`